lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs

```diff
@@ -280,13 +301,158 @@ impl Client {
                 self.pending_sd_info_resp_handlers.insert(
                     rdp_req.device_io_request.completion_id,
                     Box::new(
-                        |_cli: &mut Self,
+                        |cli: &mut Self,
                          res: SharedDirectoryInfoResponse|
                          -> RdpResult<Vec<Vec<u8>>> {
-                            let _rdp_req = rdp_req;
+                            let rdp_req = rdp_req;
                             debug!("got {:?}", res);
-                            // TODO(isaiah): see https://github.com/FreeRDP/FreeRDP/blob/511444a65e7aa2f537c5e531fa68157a50c1bd4d/channels/drive/client/drive_file.c#L207
-
+                            if res.err_code == 0 {
+                                // The file exists
+                                // https://github.com/FreeRDP/FreeRDP/blob/511444a65e7aa2f537c5e531fa68157a50c1bd4d/channels/drive/client/drive_file.c#L214
+                                if res.fso.file_type == 1 {
```

Comment:
What does `file_type` `1` mean? Would a named constant or enum make this clearer?

Reply:
Yes definitely, that's addressed in https://github.com/gravitational/teleport/pull/12750 ([here](https://github.com/gravitational/teleport/pull/12750/files#diff-04d4ba71603f3d244046f417a7b96c4830840c73c3d18a361b83364095ee2f3cR1065-R1068))